paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/log/LogStoreTableFactory.java

```diff
@@ -50,7 +52,7 @@
  * <p>Log tables are for processing only unbounded data. Support streaming reading and streaming
  * writing.
  */
-public interface LogStoreTableFactory extends DynamicTableFactory {
+public interface LogStoreTableFactory extends DynamicTablePaimonFactory {
```

Comment:
Maybe we don't need `DynamicTablePaimonFactory` and `PaimonFactory`? Just a `LogStoreTableFactory` with `String factoryIdentifier()` is OK?

Reply:
yes we only need  String factoryIdentifier() ;